WordPress/WordPressTest/ViewRelated/Post/Views/PostCardStatusViewModelTests.swift

```diff
@@ -0,0 +1,85 @@
+
+@testable import WordPress
+
+import Nimble
+
+private typealias ButtonGroups = PostCardStatusViewModel.ButtonGroups
+
+class PostCardStatusViewModelTests: XCTestCase {
+    private var contextManager: TestContextManager!
+    private var context: NSManagedObjectContext!
+
+    override func setUp() {
+        super.setUp()
+        contextManager = TestContextManager()
+        context = contextManager.newDerivedContext()
+    }
+
+    override func tearDown() {
+        context = nil
+        contextManager = nil
+        super.tearDown()
+    }
+
+    func testExpectedButtonGroupsForVariousPostAttributeCombinations() {
+        // Arrange
+        print(String(describing: PostCardStatusViewModel.Button.edit))
```

Comment:
This `print` is necessary?

Reply:
Removed in 76e470f2929814e98eb51bcba0999d42890e2bd7